content/protocols/pusher.textile

```diff
@@ -0,0 +1,97 @@
+---
+title: Pusher Adapter
+meta_description: "Use the Pusher Adapter to migrate from Pusher to Ably by only changing your API key."
+languages:
+  - javascript
+---
+
+Ably provides the ability to migrate from Pusher to Ably using its Pusher Adapter. The protocol adapter handles all background translation and only requires you to update your API key.
```

Comment:
Ably enables migration from Pusher to Ably using its Pusher Adapter. The protocol adapter handles all background translation and only requires an API key update.